### PR TITLE
ARSSTB-170: hidden text edit, file upload rename keys, 

### PR DESCRIPTION
**Pull request by C-Thomp**
### Type of change
- [ ] Breaking change
- [x] Non-breaking change
- [x] Cosmetic change

### Description
Jira link: [ARSSTB-170](https://jira.tools.tax.service.gov.uk/browse/ARSSTB-170)

- small hidden text content change
- duplicated fileUpload messages now share the same key

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -575,13 +568,14 @@ uploadSupportingDocuments.checkYourAnswersLabel = Dogfennau ategol wedi’u huwc
...
-uploadSupportingDocuments.error.quarantine = The selected file contains a virus
-uploadSupportingDocuments.error.rejected = Mae angen i’r ffeil dan sylw fod ar ffurf pdf neu jpeg
-uploadSupportingDocuments.error.unknown = There was a problem uploading your file
+fileUpload.error.entitytoolarge = The selected file must be smaller than {0}MB
+fileUpload.error.entitytoosmall = The selected file is empty
+fileUpload.error.invalidargument = Select a file to upload
+fileUpload.error.duplicate = You can only upload distinct files
+fileUpload.error.failed = The selected file failed to upload
+fileUpload.error.quarantine = The selected file contains a virus
+fileUpload.error.rejected = Mae angen i’r ffeil dan sylw fod ar ffurf pdf neu jpeg
```
Comment (aaocs) [inline conf/messages.cy:577]: We recently did this story: 
https://jira.tools.tax.service.gov.uk/browse/ARSSTB-251
where the casing of pdf/jpg was changed to PDF/JPG. Would something similar be required for Welsh messages?

Comment (C-Thomp) [reply conf/messages.cy:577]: from discussion - this will be dealt with in a separate PR as there are more occurrences throughout messages files

**Comment by Ewan-James-Donovan** [comment]
LGTM!

**Review by Ewan-James-Donovan — APPROVED** [review]
LGTM x2

**Pull request merged**